src/app/shared/services/api.service.spec.ts

```diff
@@ -2,12 +2,7 @@ import { perosonCreditsMock, perosonCreditsMockApi } from 'src/assets/mock/perso
 import { personImageMock, personImageMockApi } from 'src/assets/mock/personImage.mock';
 import { personMockApi, personMock } from 'src/assets/mock/person.mock';
 import { movieImagesMockApi, movieImagesMock } from 'src/assets/mock/movieImages.mock';
-import { convertPersonImageApiToImage } from 'src/app/actor-page/models/converters/convertPersonImageApiToImage';
-import { convertPersonCreditsToMovie } from 'src/app/actor-page/models/converters/convertPersonCreditsToMovie';
-import { convertPersonApiToPerson } from 'src/app/actor-page/models/converters/convertPersonApiToPerson';
-import { convertImagesApiToImages } from 'src/app/movie-page/models/converters/convertImagesApiToImages';
-import { convertFulMovieInfoToMovie } from 'src/app/movie-page/models/converters/convertFullMovieInfoToMovie';
-import { convertApiResultToResult } from 'src/app/main/models/converters/convertApiResultToResult';
+import * as converts from 'src/assets/mock/index-converters';
```

Comment:
I recommend import in curly braces because build tools will be able to optimize imports to speedup loading + will be able to remove unused ones